about_our_class.md

```diff
@@ -8,6 +8,14 @@ I like to go sailing on the weekends, but my true passion is collecting dryer li
  * Especially if my IDE supports previewing markdown (Visual Studio Code does)
  * If you're unfamiliar with Markdown, now's a great time to try things out!
 
+## John Lorette III
+Hi, my name is John Lorette III.
+I have a variety of hobbies and/or 'passions':
+ * Video Games
+ * Fiddling with Electronic Hardware and Software/Code
+ * Playing the Piano
+ * Reading and Learning if possible
+ * Eating Delicious Food
  ## Jaron Ritter
```

Comment:
Formatting-wise, it's good to have blank lines between different sets of text like this (just for readability when editing)